docker/Dockerfile.backend

```diff
@@ -35,6 +35,11 @@ RUN wget -O /app/vm_deps/gcloud-sdk.tar.gz https://dl.google.com/dl/cloudsdk/cha
     ./google-cloud-sdk/bin/gcloud components install beta cloud-datastore-emulator app-engine-python app-engine-python-extras && \
     rm gcloud-sdk.tar.gz
 
+# Install DSAdmin
+WORKDIR /app
+RUN curl -o dsadmin.tar.gz -L https://github.com/remko/dsadmin/releases/download/v0.20.0/dsadmin-v0.20.0-linux-amd64.tar.gz
```

Comment:
Try and pull the latest version instead.

Reply:
Just a note, from a security perspective, I think it might be better to pin the version. cc @U8NWXD

Reply:
I agree. We should always pin versions, both for security and to make debugging easier